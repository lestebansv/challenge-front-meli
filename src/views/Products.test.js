import { render } from "@testing-library/react";
import Products from "./products";

jest.mock('react-router-dom', () => ({
    ...jest.requireActual('react-router-dom'),
    useLocation: () => {
        return {
            pathname: '/items',
            state: {},
            key: '',
            search: 'kindle',
            hash: '',
        }
    }
}));

describe("Products", () => {

    it('renders', async () => {
        const categories = ['Computaci√≥n'];

        const { container } = render(<Products data={categories} />);

        expect(container).toMatchSnapshot();
    });
});