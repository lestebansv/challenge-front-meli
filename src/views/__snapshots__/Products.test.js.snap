// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Products renders 1`] = `<div />`;

exports[`Products validate render view 1`] = `<div />`;
